src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkTests.ps1

```diff
@@ -218,23 +218,19 @@ function Test-VirtualNetworkCRUDWithDDoSProtection
         Assert-AreEqual "10.0.1.0/24" $expected.Subnets[0].AddressPrefix
         Assert-AreEqual true $expected.EnableDDoSProtection
         Assert-AreEqual $ddosProtectionPlan.Id $expected.DdosProtectionPlan.Id
-        Assert-AreEqual false $expected.EnableVmProtection
```

Comment:
Why are you removing these asserts?

Reply:
We have removed this functionality. You can see that we also removed " $expected.EnableVmProtection = $true"

Reply:
Is this a breaking change in PowerShell?

Reply:
nope

Reply:
How are you removing functionality without making a breaking change? Could users previously use this property and now can't? Can you give a little more background on when/how this was changed? Sorry, want to be sure that we don't break any existing customers.